client/src/components/Channel.js

```diff
@@ -168,9 +168,9 @@ class Channel extends React.Component {
       ChannelActions.sendMessage(this.state.channelName, text);
   }
 
-  sendFile(filePath: string, buffer) {
+  sendFile(filePath: string, buffer, mimeType = 'application/octet-binary') {
```

Comment:
Definitely prefer the proposed `meta` field. Added it to ipfs-post (https://github.com/haadcode/ipfs-post/commit/12248985c60c65953e80c3600312f8b8b5d58694).